atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/InstrumentedCassandraClient.java

```diff
@@ -68,14 +67,14 @@ public void batch_mutate(String kvsMethodName,
             });
         });
 
-        tablesToCells.forEach((table, numberOfCells) -> getCellsWrittenMeterForTable(table).mark(numberOfCells));
+        tablesToCells.forEach((table, numberOfCells) -> updateCellsWrittenMeterForTable(table, numberOfCells));
     }
 
-    private Meter getCellsWrittenMeterForTable(String table) {
-        return taggedMetricRegistry.meter(
-                MetricName.builder()
+    private void updateCellsWrittenMeterForTable(String table, Long numberOfCells) {
+        taggedMetricRegistry.counter(MetricName.builder()
```

Comment:
I do wonder if we should cache the table->counter mapping to save on constructing the MetricName every time.

Reply:
That's what TaggedMetricRegistry does.

Reply:
I meant avoiding having to do the `MetricName.builder()...build()` all the time and then having to has all the parts vs just a straight lookup.  But I agree the perf difference is probably negligible.